src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPicker.cs

```diff
@@ -1069,19 +1069,19 @@ private void OnHexTextChanging(TextBox sender, TextBoxTextChangingEventArgs args
 		private Rgb GetRgbColorFromTextBoxes()
 		{
 			// Uno Doc: There is no drop-in C# equivalent for the C++ '_wtoi' function; therefore, this code is re-written.
-			_ = int.TryParse(m_redTextBox?.Text, out int redValue);
-			_ = int.TryParse(m_greenTextBox?.Text, out int greenValue);
-			_ = int.TryParse(m_blueTextBox?.Text, out int blueValue);
+			_ = int.TryParse(m_redTextBox?.Text, CultureInfo.InvariantCulture, out int redValue);
+			_ = int.TryParse(m_greenTextBox?.Text, CultureInfo.InvariantCulture, out int greenValue);
+			_ = int.TryParse(m_blueTextBox?.Text, CultureInfo.InvariantCulture, out int blueValue);
 
 			return new Rgb(redValue / 255.0, greenValue / 255.0, blueValue / 255.0);
 		}
 
 		private Hsv GetHsvColorFromTextBoxes()
 		{
 			// Uno Doc: There is no drop-in C# equivalent for the C++ '_wtoi' function; therefore, this code is re-written.
-			_ = int.TryParse(m_hueTextBox?.Text, out int hueValue);
-			_ = int.TryParse(m_saturationTextBox?.Text, out int saturationValue);
-			_ = int.TryParse(m_valueTextBox?.Text, out int valueValue);
+			_ = int.TryParse(m_hueTextBox?.Text, CultureInfo.InvariantCulture, out int hueValue);
+			_ = int.TryParse(m_saturationTextBox?.Text, CultureInfo.InvariantCulture, out int saturationValue);
+			_ = int.TryParse(m_valueTextBox?.Text, CultureInfo.InvariantCulture, out int valueValue);
```

Comment:
@Youssef1313 This may be incorrect as the value is read from the user's input. Integers are not a big deal still, as they don't vary much, but it's best to parse using the correct culture.

Reply:
@jeromelaban My thought was that the original C++ code isn't aware of CurrentCulture at all.

Reply:
I see. Maybe integers are not a concern at all then. Let's keep it as is.